app/templates/send_sms.html

```diff
@@ -0,0 +1,41 @@
+{% extends "_base_page.html" %}
+{% block page_title %}
+GOV.UK Notify
+{% endblock %}
+
+{% block content %}
+<div class="page-container">
+    {%
+    with
+    heading = "Send an SMS"
+    %}
+    {% include "toolkit/page-heading.html" %}
+    {% endwith %}
+
+
+    <form autocomplete="off" action="" method="post">
+        <div class="grid-row">
+            <div class="column-two-thirds">
+                <div class="question" id="mobile">
+                    <label for="mobile">
```

Comment:
`for` needs to match `id` of the form element (eg for this it would be `input-name`).